components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/commons/js/utils/utils.js

```diff
@@ -17,102 +18,109 @@ define(['require', 'jquery','constants'],
+                            },
+                            "type": "BUNDLE",
+                            "lookupRegex": "kafka-clients-(.+).jar"
+                        },
+                        {
+                            "name": "redis-server",
+                            "version": "2.3.0",
+                            "download": {
+                                "info": {
+                                    "description": "redis clients is like a one extension",
+                                    "install": "by using this way we can install redis-clients"
+                                },
+                                "autoDownloadable": false,
+                                "url": "https://repo1.maven.org/maven2/org/apache/redis/redis_2.11/2.1.1/kafka_2.11-2.1.1.jar"
+                            },
+                            "type": "BUNDLE",
+                            "lookupRegex": "kafka-server-(.+).jar"
+                        }
+                    ]
+                },
+                "grpc": {
+                    "extensionStatus": "NOT_INSTALLED",
+                    "extensionInfo": {
+                        "name": "grpc",
+                        "version": "3.1.2-SNAPSHOT"
+                    },
+                    "manuallyInstall": [
+                        {
+                            "name": "grpc-clients",
+                            "version": "2.3.0",
+                            "download": {
+                                "info": {
+                                    "description": "grpc clients is like a one extension",
+                                    "install": "by using this way we can install grpc-clients"
+                                },
+                                "autoDownloadable": false,
+                                "url": "https://repo1.maven.org/maven2/org/apache/grpc/grpc_2.11/2.1.1/grpc_2.11-2.1.1.jar"
+                            },
+                            "type": "BUNDLE",
+                            "lookupRegex": "grpc-clients-(.+).jar"
+                        },
+                        {
+                            "name": "grpc-server",
+                            "version": "2.3.0",
+                            "download": {
+                                "info": {
+                                    "description": "grpc server is like a one extension",
+                                    "install": "by using this way we can install grpc-server"
+                                },
+                                "autoDownloadable": false,
+                                "url": "https://repo1.maven.org/maven2/org/apache/grpc/grpc_2.11/2.1.1/grpc_2.11-2.1.1.jar"
+                            },
+                            "type": "BUNDLE",
+                            "lookupRegex": "grpc-server-(.+).jar"
+                        }
+                    ]
+                },
```

Comment:
@senthuran16 please check whether json config is correct

Reply:
Yes, this json config is correct.